data/json/recipes/recipe_ammo.json

```diff
@@ -612,34 +612,33 @@
   "subcategory": "CSC_AMMO_COMPONENTS",
   "skill_used" : "fabrication",
   "skills_required": [[ "archery", 5 ], [ "mechanics", 3 ]],
-  "difficulty" : 7,
+  "difficulty": 6,
   "time" : 5000,
   "book_learn": [[ "textbook_anarch", 4 ] , ["recipe_bullets" , 7]],
   "using": [ [ "soldering_standard", 10 ] ],
   "qualities":[
-    {"id":"HAMMER","level":1},
-    {"id":"SCREW","level":1}
+    {"id":"SAW_M","level":1}
   ],
   "components" : [
     [
-      [ "scrap", 2 ]
+      [ "can_food_unsealed", 1 ],
+      [ "can_drink_unsealed", 1 ],
+      [ "canister_empty", 1 ]
     ],
     [
       [ "superglue", 1 ],
       [ "string_36", 1 ]
     ],
     [
-      [ "can_food_unsealed", 1 ],
-      [ "can_drink_unsealed", 1 ],
-      [ "canister_empty", 1 ]
-    ],
-    [
-      [ "nail", 20 ],
-      [ "bb", 40 ]
+        ["smpistol_primer", 1 ],
+        ["lgpistol_primer", 1],
+        ["smrifle_primer", 1],
+        ["lgrifle_primer", 1],
+        ["shotgun_primer", 1]
     ],
     [
-      [ "gunpowder", 60 ],
-      [ "chem_black_powder", 60 ]
+      [ "gunpowder", 20 ],
+      [ "chem_black_powder", 40 ]
```

Comment:
Every other recipe uses the same amount of blackpowder as gunpowder, because a single unit of blackpowder weighs 5 times as much as a single unit of smokeless powder.

Reply:
Odd, as the explosives recipes (which I started editing) used the double values. Which probably means we will have to make a decision on what the standard will be. (No idea how easy the black powder is to get btw, which probably should influence the decision, and not sure if we shoudn't also change the weight of the gunpowder/blackpowder).

Reply:
Well, I can easily change it the other way around.

And you are right, to make 50 units of black gunpowder you need to perform one of these recipes: http://cdda-trunk.chezzo.com/chem_black_powder/craft doesn't take much time, but does take quite a few resources. So you are right on that.

Reply:
Updated the amounts. Thanks for mentioning it. I had a bad assumption on those amounts.

Reply:
No worries.